src/create.js

```diff
@@ -53,7 +54,23 @@ function newThingUI (createContext, dataBrowserContext, thePanes) {
     pre.appendChild(dom.createTextNode(message))
   }
 
-  var selectNewTool = function (_event) {
+  function styleTheIcons (style) {
+    for (var i = 0; i < iconArray.length; i++) {
```

Comment:
Why not use `iconArray.forEach` instead of the for loop? You have to refer to the items using `iconArray[i]` anyway, so would make the code easier to read.

I would in fact take it a step further, and use `.filter` method as well:

```
function styleTheIcons (style) {
  const dimmedIconStyle = `${iconStyle}${style}opacity: 0.3;`
  iconArray
    .filter(icon => icon.disabled)
    .forEach(icon => icon.setAttribute('style', dimmedIconStyle))
}
```